src/test/java/seedu/duke/commands/TaskCommandTest.java

```diff
@@ -76,20 +75,7 @@ void testMarkTask() {
         CommandResult commandResult = command2.execute();
         assertEquals("Task marked successfully: [T][X] Read book", commandResult.getFeedbackToUser());
     }
-
-    @Test
-    void testPrintTask(){
-        TaskList tasks = new TaskList();
-        AddTaskCommand command = new AddTaskCommand("todo", "Read book");
-        command.setData(tasks);
-        command.execute();
-
-        ListTaskCommand listTaskCommand = new ListTaskCommand();
-        listTaskCommand.setData(tasks);
-        CommandResult commandResult = listTaskCommand.execute();
-        assertEquals("Here are the tasks in your list!\n1. [T][ ] Read book\n", 
-            commandResult.getFeedbackToUser());
-    }
+    
```

Comment:
Maybe we can use dummy data in test json file to have static print as we want to increase our coverage. But this is minor.